src/gtk/window.cpp

```diff
@@ -1910,13 +1960,8 @@ gtk_window_button_release_callback( GtkWidget *WXUNUSED(widget),
     event.SetEventObject( win );
     event.SetId( win->GetId() );
 
-    // We ignore the result of the event processing here as we don't really
-    // want to prevent the other handlers from running even if we did process
-    // this event ourselves, there is no real advantage in doing this and it
-    // could actually be harmful, see #16055.
-    (void)win->GTKProcessEvent(event);
-
-    g_lastMouseEvent = nullptr;
+    if ( win->GTKProcessEvent(event) )
+        return TRUE;
 
     return FALSE;
```

Comment:
```suggestion
    return win->GTKProcessEvent(event);
```

Reply:
I wanted to minimize changes to the existing code and, also, we are converting `bool` to `gboolean` explicitly in other places, so it would need to be

```suggestion
    return win->GTKProcessEvent(event) ? TRUE : FALSE;
```

which is not great and I'd rather to leave it as is, unless you feel really strongly about it.